resources/extruders/skriware_2_extruder_0.def.json

```diff
@@ -0,0 +1,19 @@
+{
+    "id": "skriware_2_extruder_0",
```

Comment:
These IDs are superfluous. They aren't being used. Could you remove them?

Reply:
Are the files good to go or still need more changes?